src/components/consultation-pad-contents/consultation-pad-contents.scss

```diff
@@ -1,24 +1,29 @@
 .saveButton{
-  left: 30%
+  min-height: 0;
+  position: fixed;
+  right: 1rem;
+  padding: 0.4rem 1.5rem 0.4rem 1.5rem;
 }
-
 .microphoneIcon {
   fill: rgb(240, 18, 18);
-  height: 55px;
-  width: 50px;
+  height: 2rem;
+  width: 2rem;
+  cursor: pointer;
 }
-
 .stopIcon {
   fill: rgb(240, 18, 18);
-  height: 55px;
-  width: 75px;
-}
-
+  height: 2rem;
+  width: 2rem;
+  cursor: pointer;
+  }
 .padBottomArea{
-    display: flex;
-    padding-left: 1rem;
-    align-items: center;
-    justify-content: center;
+  display: flex;
+  justify-content:center;
+  padding-top: 1rem;
   }
-
+h6{
```

Comment:
Giving css on tag name is not a good idea: 
https://frontstuff.io/you-need-to-stop-targeting-tags-in-css
Assign a class and then try to add css.